portfolio/src/main/java/com/google/sps/servlets/randomGet.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns HTML that contains the page view count. */
+@WebServlet("/random")
```

Comment:
Same note about specificity, maybe this address can be "/random-number"

Reply:
check